fix(cloud): resolve edge plugin sources in cloud API typecheck

## Problem\n\nOn current `develop` (`0b5e84ca5869`), a clean `bun install --frozen-lockfile && bun run verify` can fail in `@elizaos/cloud-api#typecheck` with `TS2307: Cannot find module '@elizaos/plugin-web-search/edge'`. The API tsconfig maps `@elizaos/cloud-shared` directly to shared source, so TypeScript follows `shared-eliza-runtime.ts` but does not inherit the shared project's path aliases. It then depends on whether plugin edge declarations happen to have been built by an unrelated concurrent task.\n\nThe same direct source path also imports `@elizaos/plugin-scheduling/edge`; it can succeed incidentally when another task builds scheduling first.\n\n## Acceptance criteria\n\n- Cloud API typechecking resolves both edge imports directly from source.\n- A clean repository verify no longer depends on incidental Turbo scheduling for these modules.\n- Focused cloud API typecheck and root verify pass.

--- a/packages/cloud/services/container-control-plane/tsconfig.json
+++ b/packages/cloud/services/container-control-plane/tsconfig.json
@@ -23,10 +23,16 @@
       "@elizaos/core/*": ["../../../../packages/core/src/*"],
       "@elizaos/logger": ["../../../logger/src/index.ts"],
       "@elizaos/logger/*": ["../../../logger/src/*"],
+      "@elizaos/plugin-scheduling/edge": [
+        "../../../../plugins/plugin-scheduling/src/edge.ts"
+      ],
       "@elizaos/plugin-sql": [
         "../../../../plugins/plugin-sql/src/index.node.ts"
       ],
       "@elizaos/plugin-sql/*": ["../../../../plugins/plugin-sql/src/*"],
+      "@elizaos/plugin-web-search/edge": [
+        "../../../../plugins/plugin-web-search/src/edge.ts"
+      ],
       "drizzle-orm": ["../../../node_modules/drizzle-orm"],
       "drizzle-orm/*": ["../../../node_modules/drizzle-orm/*"],
       "@/*": ["../../../*"]

